dspace/modules/api/src/main/java/org/dspace/curate/TransferToDash.java

```diff
@@ -0,0 +1,364 @@
+                    // TODO add affiliation
+                    allAuthors.add(authorObj);
+                }
+                vals = item.getMetadata("dc.contributor.author");
+                for(int i = 0; i < vals.length; i++) {
+                    String authorName = vals[i].value;
+                    ObjectNode authorObj = objectMapper.createObjectNode();
+                    int comma = authorName.indexOf(",");
+                    authorObj.put("firstName", authorName.substring(comma+2));
+                    authorObj.put("lastName", authorName.substring(0, comma));
+                    // TODO add affiliation
+                    allAuthors.add(authorObj);
+                }
+                dataset.put("authors", allAuthors);
+
+		log.info(handle + " done.");
+	    } catch (Exception e) {
+		log.fatal("Skipping -- Exception in processing " + handle, e);
+		setResult("Object has a fatal error: " + handle + "\n" + e.getMessage());
+		report("Object has a fatal error: " + handle + "\n" + e.getMessage());
+		
+		context.abort();
+		return Curator.CURATE_SKIP;
+	    }
+
+            // write this item to json
+            objectMapper.writerWithDefaultPrettyPrinter().writeValue(new File("/tmp/transferToDash.json"), dataset);
+
+            // send this item to the server
+            sendToDash(objectMapper.writeValueAsString(dataset));
+            
+            // provide output for the console
+            setResult("Last processed item = " + handle + " -- " + packageDOI);        
+            report(handle + ", " + packageDOI + ", " + articleDOI + ", \"" + journal + "\", " +
+                   numberOfFiles + ", " + packageSize);
+
+	} else {
+	    log.info("Skipping -- non-item DSpace object");
+	    setResult("Object skipped (not an item)");
+	    context.abort();
+	    return Curator.CURATE_SKIP;
+        }
+
+	// slow this down a bit so we don't overwhelm the production SOLR server with requests
+	try {
+	    Thread.sleep(20);
+	} catch(InterruptedException e) {
+	    // ignore it
+	}
+
+	log.debug("TransferToDash complete");
+
+	try { 
+	    context.complete();
+        } catch (SQLException e) {
+	    log.fatal("Unable to close database connection", e);
+	}
+	return Curator.CURATE_SUCCESS;
+    }
+
+    private void sendToDash(String jsonString) {
+        BufferedReader reader = null;
+        try {
+            URL url = new URL(dashServer + "/api/datasets");
+            HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+            //connection.setRequestProperty("Content-Type", "application/json; charset=UTF-8");
+            connection.setRequestProperty("Content-Type", "application/json");
+            connection.setRequestProperty("Accept", "application/json");
+            connection.setRequestProperty("Authorization", "Bearer " + oauthToken);
+            connection.setDoOutput(true);
+            connection.setRequestMethod("POST");
```

Comment:
FYI: I'm trying to parse this out with @sfisher right now, but apparently he and @dmoles think that if you want to create a new entry that specifies the identifier (like with a specific DOI), you should do a PUT. I don't agree with this behavior, but that's what it's doing right now.